src/main/java/step02/NumberRule.java

```diff
@@ -0,0 +1,17 @@
+package step02;
+
+import java.util.List;
+import java.util.stream.Collectors;
+import java.util.stream.IntStream;
+
+import static step02.LottoConfig.*;
+
+public abstract class NumberRule {
+    List<Integer> integers = IntStream
```

Comment:
상속 받은 곳에서 재 할당하지 못하도록 그리고 한번만 초기화 되도록 `static` `final`로 선언 되면 좋을 것 같습니다 😄 
```suggestion
    static final List<Integer> INTEGERS = IntStream
```